backend/app/models/ShortLink.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+
+const adminSchema = new Schema(
```

Comment:
Please use better naming conventions. This could be misleading.
Please refactor it to shortLinksSchema